Sming/Libraries/MPU6050/README.rst

```diff
@@ -0,0 +1,20 @@
+MPU6050 Gyro / Accelerometer
+=============================
+
+   :members:
+
+# MPU6050 Six-Axis (Gyro + Accelerometer)
+Based on code from `jrowberg/i2cdevlib <https://github.com/jrowberg/i2cdevlib/tree/master/ESP32_ESP-IDF/components/MPU6050>` @ 605a740. Most of the code is the same, except:
+
```

Comment:
Almost... remove `:members:` and line starting '#'. Also the URL requires __ at the end!

```
Based on code from `jrowberg/i2cdevlib <https://github.com/jrowberg/i2cdevlib/tree/master/ESP32_ESP-IDF/components/MPU6050>`__ @ 605a740. Most of the code is the same, except:
```